docs/static-types.md

```diff
@@ -31,6 +31,34 @@ If we try to call this method with the wrong parameter type, we get the expected
 ```scala mdoc:fail
 string.count(123)
 ```
+### Scala methods representing bracket access
+The annotation @PyBracketAccess can be used on methods to mark them as representing bracket access on an object. The target method must have one (to read the value) or two parameters (to update the value). So, for example, we have a static facade for a list of integers:
```

Comment:
Need backticks around `@PyBracketAccess`

Reply:
Also, can have a better explanation than `representing bracket access on an object`. Perhaps `representing indexing into the Python object using brackets in Python syntax`?

Reply:
Also, to make the docs more active, we can reword `So, for example, we have` to `For example, we can create`.